Cargo.toml

```diff
@@ -13,7 +13,7 @@ categories = ["algorithms", "database-implementations"]
 
 [dependencies]
 log = "0.4.1"
-protobuf = "2.0"
+protobuf = "1.7"
```

Comment:
`eraftpb.rs` generated by `rust-protobuf:1.7` still has the problem. `read_proto3_enum_with_unknown_fields_into` is still called in `eraftpb.rs`. So maybe we can generate the file by `cargo install protobuf --vers=1.4.1 --force`.